static/app/actionCreators/guides.tsx

```diff
@@ -56,6 +62,29 @@ export function recordFinish(guide: string, orgId: string | null) {
     },
   });
 
+  let tour;
+  switch (guide) {
+    case 'sidebar_v2':
+      tour = 'tabs';
+      localStorage.removeItem('sidebarGuide');
```

Comment:
I haven't used it yet but it seems that we could use the `useLocalStorageState` hook here 🧐

Reply:
@priscilawebdev I don't think that works if you are relying on sharing state between components that have separate `useLocalStorageState` invocations but I could be wrong